config/rbac/role.yaml

```diff
@@ -23,6 +23,12 @@ rules:
   - secrets
   verbs:
   - '*'
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  verbs:
+  - '*'
```

Comment:
https://github.com/kyma-project/btp-manager/pull/180 looks operational, there is a newly added `ServiceAccount` and it generated RBAC for it